### DISCUSSION TITLE
Skip publishing to App Store Connect: no IPAs or PKGs found

### DISCUSSION THREAD

**Discussion by Baptmix** [codemagic-ci-cd/codemagic-docs#1296] [Q&A] [answered]
Hello,

I use CodeMagic for CI/CD on an Ionic application with Capacitor and VueJS. The whole build process goes very well, up to the publishing process. I have this final log that makes the app not be sent to TestFlight : 

Skip publishing to App Store Connect: no IPAs or PKGs found

ID : 63b942a48ab154742fe6b4d9

I have the codemagic.yml configuration for Ionic and Capacitor... And I have these lines :

```
    artifacts:
      - build/ios/ipa/*.ipa
```
![image](https://user-images.githubusercontent.com/56630992/211145195-f9233bce-34d0-40a2-86ae-23bf2b936a52.png)

![image](https://user-images.githubusercontent.com/56630992/211145181-55a2ec19-5a23-417e-b416-f3a994bb935a.png)

Thanks in advance

**Comment by K-Codemagic** [comment, marked as answer]
Hi, I see the latest build is successfully published. Can you please confirm it?

**Comment by Baptmix** [comment]
Hi, yes it's good for me. Thanks :)

**Comment by timvanuum** [comment]
I'm having the same problem, what was the fix?

**Reply by himesh-cm** [reply]
Hi @timvanuum, can you please open another case for discussion and share the details with your build id so I can look at it?

**Reply by timvanuum** [reply]
Thanks @himesh-cm, I made a new discussion

**Comment by Baptmix** [comment]
Hello,

I fixed myself this bug by changing :
`    artifacts:
      - build/ios/ipa/*.ipa`

by :
`    artifacts:
      - $CM_BUILD_DIR/ios/App/build/ios/**/*.ipa`

Sincerely,

**Reply by timvanuum** [reply]
Thank you so much! :smile: Works now!

**Reply by Baptmix** [reply]
> Thank you so much! 😄 Works now!

Perfect ! With pleasure ! 🙂
